### DISCUSSION TITLE
Llama packs LLM

### DISCUSSION THREAD

**Discussion by pjbruno327** [run-llama/llama_index#11712] [Q&A]
How can the embedding model and LLM be changed when using a llama index pack?

**Comment by JT-McLeod** [comment]
You could try:

from llama_index.core import Settings

Settings.llm = Ollama(model="llama2", request_timeout=700.0)
Settings.embed_model = resolve_embed_model("local:BAAI/bge-small-en-v1.5")

Though I am still hitting a call to OpenAI_API from something else.
